### ISSUE TITLE
No email from rhub

### ISSUE THREAD

**Issue by rolfTurner** [r-hub/rhub#408]
I am trying to build a Windoze binary of a package using rhub.  I invoked the check() function and after a few minutes got a message on screen indicating that the package had passed the checks without problem.

In the past when I have done this I have almost immediately received an email which provided a link from which  I could download the Windoze binary.  This time, nada.

Has something changed?  (It's been a while since I used rhub.)  Or has something gone wrong with the system?

**Comment by maelle** [comment]
No email even in the spam folder, right?

**Comment by rolfTurner** [comment]
On Mon, 7 Sep 2020 07:37:15 +0000
Maëlle Salmon <notifications@github.com> wrote:

> No email even in the spam folder, right?

Well, not as such. :-)  But that was essentially the problem.  The U.
of Auckland (which operates the mail server that I use) has a spam
blocking facility.  They send you an email when messages get blocked,
and you can then log in to a "portal" and release these messages --- or
not, as the case may be.

I had not noticed that I'd got an email from this spam blocking
facility.  (Blush!!!) I did notice this last night, but had not yet got
around to updating the issue that I had raised on rhub.  (Sorry 'bout
that.)

Bottom line:  rhub is working just fine.  Apologies for the noise.

cheers,

Rolf Turner

-- 
Honorary Research Fellow
Department of Statistics
University of Auckland
Phone: +64-9-373-7599 ext. 88276
